lib/membrane/rtp/packet.ex

```diff
@@ -14,6 +14,9 @@ defmodule Membrane.RTP.Packet do
   @enforce_keys [:header, :payload]
   defstruct @enforce_keys
 
+  # 0xBEDE for Venerable Bede from RFC 8285
+  @one_byte_header_identifier <<190::8, 222::8>>
```

Comment:
Can you simply use hex values? I use that inside `Membrane.RTP.SilenceDiscarder`.